src/main/kotlin/co/yappuworld/team/client/application/AdminTeamService.kt

```
@@ -0,0 +1,290 @@
+
+    private fun createTeamMembers(
+        team: TeamEntity,
+        activityUnitIds: List<UUID>
+    ) {
+        val teamMembers = activityUnitIds.map { activityUnitId ->
+            TeamMemberEntity(team = team, activityUnitId = activityUnitId)
+        }
+
+        try {
+            teamMemberCommandService.saveAll(teamMembers)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)